### DISCUSSION TITLE
Census

### DISCUSSION THREAD

**Discussion by claugp06** [perealberto/decide-single-zambrano#38] [Ideas]
Una duda/pregunta ya que tenemos log in creado se puede limitar sencillamente que solo admin pueda descargar el export census o que solo pueda crear un censo el admin desde el frontend. ¿Espero que @laurolmer haga el merge y creo una rama de fix para ello o como lo veis mejor?

**Comment by laurolmer** [comment]
yo veo bien una respectiva rama Fix para implementar ese arreglo.

**Comment by claugp06** [comment]
Perfecto,así lo he implementado
